travis-script.sh

```diff
@@ -40,7 +40,7 @@ if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
     fi
   else
     echo "Building feature branch"
-    mvn verify --settings ${MVN_SETTINGS} -B -V -PwithMutationTests,rpm,deb
+    mvn install --settings ${MVN_SETTINGS} -B -V -PwithMutationTests,rpm,deb
```

Comment:
Why do you need to install the artifacts from a feature branch ? Is it related to the `$SITE_BRANCH_NAME` variable used in `.travis.yml` ? Is there a requirement to have the artifacts deployed locally to create the maven site ? If that's the case, we need to decide if we want to publish maven site for feature branches.